src/Microsoft.DotNet.ExecutablePackageObtainer/DotnetToolsConfig.cs

```diff
@@ -0,0 +1,55 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
Generated deserializer. I find that is the most easy way to change if we want to change the xml format again

Reply:
What are the steps for changing the format and regenerating? Maybe I'm just missing it, but they don't seem to be included in the change anywhere. We shouldn't have generated files with no automation to regenerate.

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/serialization/xml-schema-def-tool-gen

It is a "standard" but old way to generate such file, however, it never built in nicely with Visual Studio or being part of the automation.

I don't know what is the "current" way of taking information out of XML. I could do it by selecting specific nodes, just like JSON. However the code could end up just like the generated code.

Reply:
If automating is hard, consider just checking in a file with two properties and no scary notices about not editing it. We can evolve to automated generation from schema if the schema ever actually gets non-trivial.

Reply:
I'm aware of the xsd tool, but I don't see what args I'd need to pass to regenerate it documented in your change. I'm generally averse to files that say "auto-generated do not edit" but the process to generate them is manual, undocumented invocation of a tool. What I've seen in the past is that folks just ignore the comment, make manual edits, and the first person to actually try to re-use the tool ends up stuck with changes getting reverted by the tool. 

Do you think this schema is going to change often? It's so tiny. I'd just reformat this code to modern style (it'd be just a few lines with auto-properties). Updating that to add a new property by just typing it in would be faster than figuring out how to run the tool. If the schema really does get too complex to hand-author, then we can revisit.

Reply:
I agree, this seems super over kill for pretty much a POCO

Reply:
I moved it to a private class, which end up to be only 3 lines...